content/chainguard/chainguard-images/comparing-images.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "How To Compare Chainguard Images with chainctl"
+linktitle: "Comparing Images"
```

Comment:
I feel like "chainctl" should be in here, because we have a lot of "comparison" language in the nav